### ISSUE TITLE
ini_file-module does not create a section, when no option/value is defined (2.0.0.2)

### ISSUE THREAD

**Issue by my2ndhead** [ansible/ansible-modules-core#2987]
##### ISSUE TYPE

Bug Report
##### COMPONENT NAME

ini_file module
##### ANSIBLE VERSION

N/A
##### SUMMARY

When a section does not already exist, it will not be created, when no options/values are defined. This was working under 1.9.

When manually creating the section and running the task, a key-value-pair of "None = none" is created.

```
name: "Creating section" 
ini_file: dest=section
          state=present
```

**Comment by ansibot** [comment]
@jpmens, @noseka1, ping. This issue is waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@jpmens, @noseka1, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by jpmens** [comment]
wontfix due to time constraints

**Comment by my2ndhead** [comment]
You can't just close this issue just because of time constraints. This is a major bug that breaks a lot of things.

**Comment by jpmens** [comment]
I agree. I did not close it.
I assume my comment caused the bot to close it. What I meant to do was to say "I cannot fix this"

@bcoca Please reopen.

**Comment by eden881** [comment]
Any updates on this? We have this problem too...

**Comment by dagwieers** [comment]
This issue was moved to ansible/ansible#37711

**Comment by dagwieers** [comment]
This repository is no longer active, so it does not get any visibility. (Except from me, I guess ;-))
So I moved this issue to https://github.com/ansible/ansible
Feel free to get things moving again at https://github.com/ansible/ansible/issues/37711
